src/pages/userInvitation/UserInvitationPageStore.js

```diff
@@ -267,7 +267,7 @@ export const useUserInvitationPageStore = defineComponentStore(
 			await createInvitation();
 
 			if (validationError.value) {
-				errors.value = validationError.value.errors;
+				errors.value = validationError.value.errors['inviteeEmail'];
```

Comment:
Hi @ipula , I checked this PR and tested on my local. I believe for this line, you intend to remove the "userId" from the list of errors coming from the API. The current code made the errors prop like this:
```
['This field is required when user id is not present.']
```
To display errors, the component expects the field id in the object in this format:
```
{
  inviteeEmail: ['This field is required when user id is not present.']
}
```

To resolve this, we can change this to:
```suggestion
                                delete validationError.value.errors['userId'];
				errors.value = validationError.value.errors;
```

Reply:
yes, my mistake I will fix this